lib/matplotlib/tests/test_contour.py

```diff
@@ -158,7 +158,7 @@ def test_contour_manual_labels():
 
 @image_comparison(baseline_images=['contour_labels_size_color'],
                   extensions=['png'], remove_text=True)
-def test_contour_manual_labels():
+def test_contour_labels_size_color():
 
```

Comment:
How are we handling regenerating images as a policy? 

See #7905 

I think we'll shortly have to revert this changed image when/if #7970 is merged.